internal/cleanup/cleanup.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package cleanup
```

Comment:
As discussed, lets rebase this on https://github.com/hashicorp/boundary/pull/3544 and move this job into that package (or a subpackage, more likely)